pkgs/development/compilers/cudatoolkit/default.nix

```diff
@@ -3,6 +3,7 @@
 , fetchurl
 , gcc7
 , gcc9
+, gcc11
```

Comment:
```suggestion
, gcc10
```
Oops, missed this

Reply:
So did I! Should be fixed in https://github.com/NixOS/nixpkgs/commit/0981d9473f646678a7279df30530c5ba51519ebe